cmd/main.go

```diff
@@ -50,23 +50,34 @@ func main() {
 	config.InitLogger(debugMode)
 	configuration := config.LoadConfig()
 
-	// Prepare repositories
-	repository := PrepareRepositories(configuration.Hedera.Validator.Db)
-
 	// Prepare Clients
-	client := PrepareClients(configuration)
+	clients := PrepareClients(configuration)
 
 	// Prepare Services
-	ethSigner := eth.NewEthSigner(configuration.Hedera.Client.Operator.EthPrivateKey)
-	contractService := bridge.NewContractService(client.ethereum, configuration.Hedera.Eth)
-	schedulerService := scheduler.NewScheduler(configuration.Hedera.Handler.ConsensusMessage.TopicId, ethSigner.Address(),
-		configuration.Hedera.Handler.ConsensusMessage.SendDeadline, contractService, client.hederaNode)
+	contractService := bridge.NewContractService(clients.ethereum, configuration.Hedera.Eth)
+	feeCalculator := fees.NewCalculator(clients.exchangeRate, configuration.Hedera, contractService)
 
-	feeCalculator := fees.NewCalculator(client.exchangeRate, configuration.Hedera, contractService)
+	apiRouter := initializeAPIRouter(feeCalculator)
 
 	// Prepare Node
 	server := server.NewServer()
 
+	if !configuration.Hedera.RestApiOnly {
```

Comment:
maybe it will be better if we log somewhere that we are starting the validator node in `REST-API Mode` only without watchers/handlers